is_core/forms/widgets.py

```diff
@@ -151,25 +151,25 @@ def _get_value(self, value):
         else:
             return value
 
-    def _get_value_display(self, value):
+    def _get_value_display(self, value, request=None):
         from is_core.utils import display_for_value
 
         value = self._get_value(value)
 
         if value in EMPTY_VALUES:
             value = EMPTY_VALUE
 
-        return display_for_value(value)
+        return display_for_value(value, request=request)
```

Comment:
Why I not find this bug earlier?

If you return queryset or model instance in the model or core method it should be rendered with links automatically